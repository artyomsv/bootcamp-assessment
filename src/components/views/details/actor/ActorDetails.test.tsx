import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import ActorDetailsView from './ActorDetails.view';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(
      <ActorDetailsView
        isFetching={false}
        fetchActorDetails={mockFn}
        clearState={mockFn}
        movies={[]}
        details={{} as any}
        navigateToMovie={mockFn}
        goBack={mockFn}
      />
    );

    expect(result).toMatchSnapshot();
  });
});
