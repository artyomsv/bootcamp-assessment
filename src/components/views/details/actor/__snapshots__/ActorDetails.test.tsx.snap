// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<withRouter(ActorDetailsView)
  classes={
    Object {
      "button": "withRouter-ActorDetailsView--button-5",
      "details": "withRouter-ActorDetailsView--details-1",
      "extendedIcon": "withRouter-ActorDetailsView--extendedIcon-6",
      "image": "withRouter-ActorDetailsView--image-2",
      "linkImage": "withRouter-ActorDetailsView--linkImage-3",
      "movies": "withRouter-ActorDetailsView--movies-4",
    }
  }
  clearState={[MockFunction]}
  details={Object {}}
  fetchActorDetails={[MockFunction]}
  goBack={[MockFunction]}
  isFetching={false}
  movies={Array []}
  navigateToMovie={[MockFunction]}
/>
`;
